test/bats/base-test.bats

```diff
@@ -26,6 +26,18 @@ SLEEP_TIME=1
 
     run kubectl run demo1 --namespace default --image=registry:5000/notation:unsigned
     assert_failure
+
+    run kubectl apply -f ./test/testdata/pod_initContainers_signed.yaml --namespace default
+    assert_success
+    
+    run kubectl apply -f ./test/testdata/pod_initContainers_unsigned.yaml --namespace default
+    assert_failure
+
+    run kubectl debug -it demo --image=registry:5000/notation:signed --target=demo
+    assert_success
+
```

Comment:
also a note that any new namespace should be cleaned up in the tear down step.

Reply:
Well noted.